### PR TITLE
Update CommentView

### LINKED ISSUE #2554: Update CommentView
info here: https://github.com/taskany-inc/bricks/issues/813

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Feature

## Related issues

Resolve #2554 

## QA Instructions, Screenshots, Recordings
  
Screenshot 2024-04-08 at 13 54 10

### CODE REVIEW

**Review by tsumo — APPROVED** [review]

File: src/components/CommentView/CommentView.tsx [hunk 1]
```diff
@@ -144,6 +144,7 @@ export const CommentView: FC<CommentViewProps> = ({
                 label: tr('Copy raw'),
                 icon: <IconClipboardOutline size="xxs" />,
                 onClick: () => notifyPromise(copyValue(descriptionRef.current), 'copy'),
+                className: null,
```
Comment (DenisVorop) [inline src/components/CommentView/CommentView.tsx:147]: Why explicitly indicate that there is no class?

File: src/components/CommentView/CommentView.tsx [hunk 2]
```diff
@@ -214,63 +221,75 @@ export const CommentView: FC<CommentViewProps> = ({
...
-                                        focused={cursor === index}
-                                        icon={item.icon}
-                                        onClick={item.onClick}
-                                    >
-                                        {item.label}
-                                    </MenuItem>
-                                )}
-                            />
+                            <Dropdown isOpen={isOpenActions} onClose={() => setIsOpenActions(false)}>
+                                <DropdownTrigger
```
Comment (DenisVorop) [inline src/components/CommentView/CommentView.tsx:241]: use services Dropdown, that you don't need local state

File: src/components/CommentView/CommentView.tsx [hunk 3]
```diff
@@ -214,63 +221,75 @@ export const CommentView: FC<CommentViewProps> = ({
...
+                                                />
+                                            }
+                                        />
+                                    )}
+                                />
+                                <DropdownPanel placement="bottom-start">
+                                    <ListView>
+                                        {dropdownItems.map((item) => (
+                                            <ListViewItem
+                                                key={item.label}
```
Comment (DenisVorop) [inline src/components/CommentView/CommentView.tsx:263]: use services DropdownPanel, you don't need ListView, cause DropdownPanel includes inside it

File: src/components/GoalContentHeader/GoalContentHeader.module.css [hunk 4]
```diff
@@ -1,16 +1,8 @@
-.Card {
-    border: 1px solid var(--layer-above);
-}
-
 /* Order of styles on page: SC, CSS Modules */
 .CardInfo.CardInfo {
```
Comment (DenisVorop) [inline src/components/GoalContentHeader/GoalContentHeader.module.css:2]: CardInfo from harmony, you don't need override style

**Review by DenisVorop — CHANGES_REQUESTED** [review]

File: src/components/CommentView/CommentView.tsx (hunk 1 above)
Comment (asabotovich) [reply src/components/CommentView/CommentView.tsx:147]: Screenshot 2024-04-10 at 15 22 26

**Review by DenisVorop — APPROVED** [review]

**Pull request merged**